developer-docs-site/docs/sdks/unity-sdk.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Unity SDK"
+slug: "unity-sdk"
+---
+
+
+# Aptos Unity SDK
+
+The [Aptos Unity SDK](https://github.com/aptos-labs/Aptos-Unity-SDK) is a .NET implementation of the [Aptos SDK](./index.md), compatible with .NET Standard 2.0 and .NET 4.x for Unity. The goal of this SDK is to provide a set of tools for developers to build multi-platform applications (mobile, desktop, web, VR) using the Unity game engine and the Aptos blockchain infrastructure.
+
+See the post [Aptos Labs brings Web3 to Gaming with its new SDK for Unity developers](https://medium.com/aptoslabs/aptos-labs-brings-web3-to-gaming-with-its-new-sdk-for-unity-developers-e6544bdf9ba9) for all of the features offered to game developers by the Aptos Unity SDK.
+
+## Core features
+
+
+- HD wallet creation and recovery
+- Account management
+    - Account recovery
+    - Message signing
+    - Message verification
+    - Transaction management
+    - Single / Multi-signer Authentication
+    - Authentication key rotation
+- Native Binary Canonical Serialization (BCS) support
+- Faucet client for devnet
+
+## User flows
+
+The Aptos Unity SDK supports these use cases:
+
+- *Progressive onboarding flow* in which users can log into a game by email. In this flow, transactions are proxied, and Aptos uses a distributed key system. The users can then onboard to a full custodial wallet if desired.
+- *In-game non-custodial wallet integration* in which game developers have the option to allow users to create full non-custodial wallets in the games.
+- *Off-game non-custodial wallet integration* in which game developers may allow users to connect to a desktop wallet or a mobile wallet within the game or create burner wallets from the parent wallet seamlessly.
+
+
+## Prerequisites
+
+### Unity versions
+| Supported Version: | Tested |
+| -- | -- |
+| 2021.3.x | ✅ |
+| 2022.2.x | ✅ |
+
+| Windows | Mac  | iOS | Android | WebGL |
+| -- | -- | -- | -- | -- |
+| ✅ | ✅ | ✅ | ✅ | ✅ |
+
+### Dependencies
+
+> As of Unity 2021.x.x, Newtonsoft Json is common dependency. Prior versions of Unity require intalling Newtonsoft.
```

Comment:
@CapCap let's make sure if we have information like this to also put it in the Unity SDK